packages/frontend/src/components/AppBar/index.tsx

```diff
@@ -42,10 +52,22 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
     setAccountMenuAnchorElement(null)
   }
 
+  // check whether user has read and closed the news drawer
+  const [localLatestTimestamp, setLocalLatestTimestamp] = useState(
+    localStorage.getItem('news-tab-latest-timestamp'),
+  )
+
+  const handleOpen = useCallback(() => {
+    // only way to update this is to change the news or clear the local storage
+    localStorage.setItem('news-tab-latest-timestamp', latestNewsTimestamp)
+    setLocalLatestTimestamp(latestNewsTimestamp)
+  }, [latestNewsTimestamp])
+
```

Comment:
why put this chunk of code here? this belongs in NewsDrawer component

Reply:
you are absolutely right, I was asking myself why too when I wrote the code, my bad...